src/controller/login.controller.ts

```diff
@@ -1,19 +1,19 @@
-import express, { Request, Response } from "express";
+import { Request, Response } from 'express';
 
-import loginService from "../service/login.service";
+import loginService from '../service/login.service';
 
 export default {
   async updateAccessTokenLogin(req: Request, res: Response) {
     try {
       const paramsId = req.params.id;
-      const userId = parseInt(paramsId);
+      const userId = parseInt(paramsId, 10);
       const newAccessToken = await loginService.updateAccessTokenLogin(
         userId,
-        req.body
+        req.body,
       );
       res.status(200).send(newAccessToken);
     } catch (error) {
-      res.status(304).send("NOT MODEFIED");
+      res.status(304).send('NOT MODEFIED');
```

Comment:
Just spelling, if we care: modefied => modified

Reply:
Yes! We should create a separate issue for this! We should do this for more files I think.